test/server/cards/02_SHD/units/ProtectorOfTheThrone.spec.js

```diff
@@ -0,0 +1,36 @@
+describe('Protector of the Throne', function() {
+    integration(function() {
+        describe('Protector of the Throne\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'protector-of-the-throne', upgrades: ['shield'] }],
+                    },
+                    player2: {
+                        groundArena: ['wampa', 'jedha-agitator'],
+                    }
+                });
+            });
+            it('should give it sentinel only as long as it is upgraded', function () {
```

Comment:
nit: newline before test case

Reply:
Great test otherwise! This covers everything we would want.

Reply:
pasted from supercommando squad 😆